v8env/src/module_loader.ts

```diff
@@ -2,25 +2,69 @@ import { sendSync } from './bridge';
 import * as fbs from "./msg_generated";
 import * as flatbuffers from "./flatbuffers"
 
-export interface ModuleInfo {
-  moduleId: string;
-  fileName: string;
-  sourceCode: string;
+export interface LoadedSourceCode {
+  isWasm: boolean;
```

Comment:
Could this use a string or enum to store the content type rather than a one off for wasm. `ModuleInfo` has a mediaType property for this which is currently inferred by filename. Maybe the module loaders in rust could return the right type when resolving so we can remove all that code from compiler.ts. Thoughts?